ThingIFSDK/ThingIFSDK/Gateway/EndNode.swift

```diff
@@ -39,7 +39,7 @@ public struct EndNode: TargetThing, Equatable {
 extension EndNode: FromJsonObject {
 
     internal init(_ jsonObject: [String : Any]) throws {
-        guard let thingID = jsonObject["thingID"] as? String,
+        guard let thingID = jsonObject["endNodeThingID"] as? String,
```

Comment:
@yongpingchen 

I wrote this according to implementation of thing-if AndroidSDK. The code is [this](https://github.com/KiiPlatform/thing-if-AndroidSDK/blob/develop/thingif/src/main/java/com/kii/thingif/ThingIFAPI.java#L720)

Is implementation of thing-if AndroidSDK is also wrong?

Moreover, in old small test, mock response of the thing id is `endNodeThingID`.
I also check specification of gateway onboarding. The sample of response describe as `endNodeThingID`. The specification is [this](https://docs.google.com/document/edit?hgd=1&id=1xNSWBeX_u2DoiIDdFvwRqP_4FQQONQnJyRQcvgUayNs#heading=h.m7ol71wbdhic)

What do you think?